docs/source/plugin/time.rst

```diff
@@ -0,0 +1,211 @@
+==========
+About time
+==========
+
+Your plugin may want to display dates and times in messages. For that, you can
+always count on the :mod:`datetime` built-in module. However, what if you would
+like to respect the date-format for a given user or a given channel? Various
+functions of :mod:`Sopel<sopel.tools.time>` can help you with that:
+
+* :func:`~sopel.tools.time.get_timezone` will fetch the right timezone for you
+* :func:`~sopel.tools.time.format_time` will format your aware datetime for you
+
+Here is a full example of that::
```

Comment:
Would it help or hurt to tell readers where they might find more in-depth code examples? 😅

```suggestion
Here is a full example of that, adapted from the built-in ``.t`` command::
```